app/backend/src/couchers/models.py

```diff
@@ -279,7 +279,7 @@ def timezone(self):
 
     @hybrid_property
     def has_completed_profile(self):
-        return self.avatar_key is not None and len(self.about_me) >= 20
+        return self.avatar_key is not None and self.about_me and len(self.about_me) >= 20
```

Comment:
if were doing self.avatar_key is not None, should we do the same for self.about_me?

Reply:
Yeah, I guess that'd be good